app/javascript/widget/views/Home.vue

```diff
@@ -1,24 +1,54 @@
 <template>
-  <div class="flex flex-1 flex-col justify-end">
-    <div class="flex flex-1 overflow-auto">
-      <!-- Load Conversation List Components Here -->
+  <div
+    class="z-50 rounded-md border-t border-slate-50 w-full"
+    :class="!portal ? 'flex flex-1 flex-col justify-end' : 'pb-2'"
+  >
+    <div v-if="portal" class="px-4 py-2 w-full">
```

Comment:
```suggestion
    <div v-if="portal && (articleUiFlags.isFetching || popularArticles.length)" class="px-4 py-2 w-full">
```

If the list of articles is empty, the UI renders a blank box. We can either show an empty state, or hide the box with the suggestion above.

<img width="389" alt="CleanShot 2023-07-27 at 11 51 59@2x" src="https://github.com/chatwoot/chatwoot/assets/18097732/8bd2c7d3-be36-472f-b5b2-0a2c602737b0">

Reply:
Missed this, will update.